andela/titan-dev-ah-frontend
# #163518647 Create Article

#### What does this PR do?
- Integrate the interface for creating an article with react 
- Enable article creation 
#### Description of Task to be completed?
- Create tests
- Connect the component with redux
- Create some articles to test the functionality. 
#### How should this be manually tested?
- Clone the repos 
- Check out the branch `ft-create-article-163518647`
- Install all dependencies by `yarn install`
- Start the server using `yarn start`
- Start the server for the backend project
- Login to the backend server using client tool like the postman
- Get the token, and store it in localStorage using `localStorage.setItem('token',value);
- Check your browser, where the application was opened
- Create an article. 
#### Any background context you want to provide?
Some of the fields that were developed in mockups were ignored since there is no backend functionality for them. Moreover, the text editor has been changed due to the adjustments issues the original one had. For reference, attached are images of mockup and real react application. 
#### What are the relevant pivotal tracker stories?
#163518647
#### Screenshots (if appropriate)

#### Questions:
